src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/controllers/BundlesRespondentControllerTest.java

```diff
@@ -0,0 +1,227 @@
+package uk.gov.hmcts.ethos.replacement.docmosis.controllers;
```

Comment:
Do we need these tests since we have API tests now for the controller? Could we do the check for errors in the API tests instead?

Reply:
Don't think it will count towards code coverage if we put in api tests. Building out the api tests could be part of the future work for this feature as not all endpoints are being hit at the moment. I'm happy to leave these checks here for now.